lms/djangoapps/program_enrollments/tests/test_models.py

```diff
@@ -0,0 +1,69 @@
+"""
+Unit tests for ProgramEnrollment models.
+"""
+from __future__ import unicode_literals
+
+from uuid import uuid4
+
+from django.test import TestCase
+
+from lms.djangoapps.program_enrollments.models import ProgramEnrollment, User
+from student.tests.factories import UserFactory
+
+
+class ProgramEnrollmentModelTests(TestCase):
+    """
+    Tests for the ProgramEnrollment model.
+    """
+    def setUp(self):
+        """
+        Set up the test data used in the specific tests
+        """
+        super(ProgramEnrollmentModelTests, self).setUp()
+        UserFactory.create()
+        self.user = User.objects.first()
+        self.email = 'foo@bar.com'
+        self.external_user_key = 'abc'
+        self.program_uuid = uuid4()
+        self.curriculum_uuid = uuid4()
+        self.status = 'enrolled'
+
+        ProgramEnrollment.objects.create(
+            user=self.user,
+            email=self.email,
+            external_user_key=self.external_user_key,
+            program_uuid=self.program_uuid,
+            curriculum_uuid=self.curriculum_uuid,
+            status=self.status
+        )
+
+        self.enrollment = ProgramEnrollment.objects.first()
```

Comment:
You can assign `self.enrollment` the result of your `create()` call on the previous line.